website/docs/index.html.markdown

```diff
@@ -85,15 +85,15 @@ If you have configuration questions, or general questions about using the provid
 
 The following arguments are supported:
 
-* `features` - (Required) A `features` block as defined below which can be used to customize the behaviour of certain Azure Provider resources.
+* `features` - (Required) A `features` block as defined below which can be used to customize the behavior of certain Azure Provider resources.
 
 * `client_id` - (Optional) The Client ID which should be used. This can also be sourced from the `ARM_CLIENT_ID` Environment Variable.
 
 * `environment` - (Optional) The Cloud Environment which should be used. Possible values are `public`, `usgovernment`, `german`, and `china`. Defaults to `public`. This can also be sourced from the `ARM_ENVIRONMENT` environment variable.
 
 * `subscription_id` - (Optional) The Subscription ID which should be used. This can also be sourced from the `ARM_SUBSCRIPTION_ID` Environment Variable.
 
-* `tenant_id` - (Optional) The Tenant ID which should be used. This can also be sourced from the `ARM_TENANT_ID` Environment Variable.
+* `tenant_id` - (Optional) The Tenant ID should be used. This can also be sourced from the `ARM_TENANT_ID` Environment Variable.
```

Comment:
this is correct
```suggestion
* `tenant_id` - (Optional) The Tenant ID which should be used. This can also be sourced from the `ARM_TENANT_ID` Environment Variable.
```